docs/howto.rst

```diff
@@ -162,29 +172,29 @@ Look back at the last command line output, from the section above. It said "Skip
 
 .. code-block:: shell
 
-    phys2bids -in tutorial_file.txt -indir /home/arthurdent/git/phys2bids/phys2bids/tests/data/ -chtrig 1 -ntp 158 -tr 1.2 -outdir /home/arthurdent/physio
+    phys2bids -in tutorial_file.txt -chtrig 1 -ntp 158 -tr 1.2 -outdir physio
 
 Now the output says:
```

Comment:
I don't see any output that follows this

Reply:
I think that will be solved by @smoia PR